docker-compose.yml

```diff
@@ -5,13 +5,15 @@ services:
   mysql:
     image: mysql/mysql-server:8.0
     restart: on-failure
-    command: ['--character-set-server=utf8mb4', '--collation-server=utf8mb4_unicode_ci']
+    command: ['--character-set-server=utf8mb4', '--collation-server=utf8mb4_unicode_ci', '--socket=/tmp/mysql.sock']
```

Comment:
curious to know what is the purpose of adding the `socket` attribute.

Reply:
When I tried to start it up I got an error of 

```
2022-05-17T17:47:03.730133Z 0 [System] [MY-013602] [Server] Channel mysql_main configured to support TLS. Encrypted connections are now supported for this channel.
2022-05-17T17:47:03.745529Z 0 [ERROR] [MY-010270] [Server] Can't start server : Bind on unix socket: Operation not permitted
2022-05-17T17:47:03.745624Z 0 [ERROR] [MY-010258] [Server] Do you already have another mysqld server running on socket: /var/lib/mysql/mysql.sock ?
2022-05-17T17:47:03.746023Z 0 [ERROR] [MY-010119] [Server] Aborting
```

This information seemed like what was needed. I wasn't running on Windows though so not sure. https://dev.mysql.com/doc/refman/8.0/en/deploy-mysql-nonlinux-docker.html

Reply:
Thanks for sharing additional details!

I did not see that error when I ran it locally on Mac with Apple chip. This should now support the Non-linux env

Reply:
Yeah, it might just be a problem with other platforms and not with M1. I'm really not sure but works fine with that there, and sounds like it would be necessary for running on Windows.